docs/quickstart.rst

```diff
@@ -0,0 +1,68 @@
+Quickstart
+==========
+
+riot is configured with a single Python file typically placed at the root of
+your project and named ``riotfile.py``.
+
+
+Here is a ``riotfile.py`` which defines 5 virtual environment instances. One
+defines how to run ``mypy``, another for ``black`` and 3 instances to run tests
+with ``pytest``::
+
+        from riot import latest, Venv
+
+        venv = Venv(
+            pys=[3.9],
+            venvs=[
+                Venv(
+                    name="fmt",
+                    command="black .",
+                    pkgs={
+                        "black": "==20.8b1",
+                    },
+                ),
```

Comment:
Reading this I'm wondering what happens if you do:

```python
venv = Venv(
            pys=[3.9],
            venvs=[
                Venv(
                    name="fmt",
                    pys=[3.8],
                    command="black .",
                    pkgs={
                        "black": "==20.8b1",
                    },
                ),
            ],
         )
```

with `pys` specified twice and in contradiction?

Reply:
The child overrides the parent value so it would result in just `pys=[3.8]`